### PR TITLE
docs: new kbd language methods for Windows Engine

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to keymanapp/keyman#3512.

### CODE REVIEW

File: developer/engine/desktop/14.0/api/IKeymanBCP47Canonicalization/GetCanonicalTag.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    'title' => "IKeymanBCP47Canonicalization::GetCanonicalTag Method"
+  ]);
+?>
+
+<h1>IKeymanBCP47Canonicalization::GetCanonicalTag Method</h1>
+
+<h2>Introduction</h2>
+
+<p>The <code>IKeymanBCP47Canonicalization::GetCanonicalTag</code> method returns a canonical tag according to data from
+<a href="">SIL langtags.json.</a>. The method will convert ISO639-3 language tags ISO639-1 language tags, remove suppressed
```
Comment (darcywong00) [inline developer/engine/desktop/14.0/api/IKeymanBCP47Canonicalization/GetCanonicalTag.php:14]: Would the link be https://github.com/silnrsi/langtags?
```suggestion
<a href="https://github.com/silnrsi/langtags">SIL langtags.json</a>. The method will convert ISO639-3 language tags ISO639-1 language tags, remove suppressed
```

File: developer/engine/desktop/14.0/api/IKeymanKeyboardLanguageInstalled2/index.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+?>
+
+<h1>IKeymanKeyboardLanguageInstalled2 Interface</h1>
+
+<h2>Introduction</h2>
+
+<p>The <code>IKeymanKeyboardLanguageInstalled2</code> interface adds extension functions for installing languages. These functions
+should be used in preference to the now-deprecated
+<a href="../IKeymanKeyboardLanguagesInstalled/Install"><code>IKeymanKeyboardLanguagesInstalled::Install</code></a> and
+<a href="../IKeymanKeyboardLanguagesInstalled/InstallByLangID"><code>IKeymanKeyboardLanguagesInstalled::InstallByLangID</code></a>
```
Comment (darcywong00) [inline developer/engine/desktop/14.0/api/IKeymanKeyboardLanguageInstalled2/index.php:16]: Should we add history to those pages when the functions were deprecated?

Comment (mcdurdin) [reply developer/engine/desktop/14.0/api/IKeymanKeyboardLanguageInstalled2/index.php:16]: Updated to reference deprecation version.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**